tests/libvmi/vmi.go

```diff
@@ -20,6 +20,7 @@
 package libvmi
 
 import (
+	. "github.com/onsi/gomega"
```

Comment:
Please avoid `.` imports. It makes it harder for a naive reviewer to understand what is pulled from `gomega` into here.
Ideally, `libvmi` should not have any `gomega` dependency. Conceptually, `libvmi` should be usable even in production code that wants to build VMs, not just in tests.